### PR TITLE
[ENG-5186] Models and db connections for migrating OSF addon credentials to GV

### PR DESCRIPTION
**Pull request by adlius**

### CODE REVIEW

File: addon_service/osf_models/models.py [hunk 1]
```diff
@@ -0,0 +1,358 @@
...
+MendeleyUserSettings.objects = MendeleyUserSettings.objects.using("osf")
+MendeleyNodeSettings.objects = MendeleyNodeSettings.objects.using("osf")
+OneDriveUserSettings.objects = OneDriveUserSettings.objects.using("osf")
+OneDriveNodeSettings.objects = OneDriveNodeSettings.objects.using("osf")
+OwnCloudUserSettings.objects = OwnCloudUserSettings.objects.using("osf")
+OwnCloudNodeSettings.objects = OwnCloudNodeSettings.objects.using("osf")
+S3UserSettings.objects = S3UserSettings.objects.using("osf")
+S3NodeSettings.objects = S3NodeSettings.objects.using("osf")
+ZoteroUserSettings.objects = ZoteroUserSettings.objects.using("osf")
+ZoteroNodeSettings.objects = ZoteroNodeSettings.objects.using("osf")
```
[the remark this answers is not in the document]
Comment (adlius) [reply addon_service/osf_models/models.py:358]: Sure. Thanks for the suggestion!

[the remark this answers is not in the document]
Comment (adlius) [reply addon_service/osf_models/models.py:353]: I tried to have a DB router but there is something wrong when it comes to fetching a relationship. I'll keep this as is for now.

File: addon_service/osf_models/fields.py [hunk 2]
```diff
@@ -0,0 +1,132 @@
...
+import jwe
+from dateutil.parser import isoparse
+from django.core.serializers.json import DjangoJSONEncoder
+from django.db import models
+from django.db.models import JSONField
+
+from app.settings import (
+    OSF_SENSITIVE_DATA_SALT,
+    OSF_SENSITIVE_DATA_SECRET,
+)
```
Comment (aaxelb) [inline addon_service/osf_models/fields.py:13]:
```suggestion
from django.conf import settings
```
django settings modules aren't meant to be imported directly

(alternately, could import `from app.env` and skip adding them to `app.settings`)

**Review by aaxelb — COMMENTED** [review]
just a couple small nits, otherwise looks good

**Pull request merged**